OpenSim/Examples/ExampleHopperDevice/exampleHopperDevice.cpp

```diff
@@ -173,8 +173,20 @@ void addDeviceConsoleReporterToModel(Model& model, Device& device,
 // the exercise. The project should execute without making any changes (you
 // should see the unassisted hopper hop slightly).
 //------------------------------------------------------------------------------
-int main()
-{
+int main(int argc, char* argv[]) {
+    //==========================================================================
+    // Command line argument parsing.
```

Comment:
Added description of arguments that can be passed. Because the two arguments are optional, listing all different ways to call the executable could be too much.